src/app/core/footer/footer.component.html

```diff
@@ -2,20 +2,19 @@
 <footer class="vertical-center align-items-center" id="footer">
   <div>
     <a class="m-4" href="https://www.facebook.com/virtrolio/" target="_blank">
-      <img src="../../../assets/images/icons/facebook.svg" alt="Our Facebook" width="32" height="32">
+      <img src="../../../assets/images/icons/facebook.svg" alt="Our Facebook" width="24" height="24">
```

Comment:
Potentially missing units on the width and height, was this intended? The same problem occurs on the instagram and email icons as well.